### DISCUSSION TITLE
Looking for an iOS/ ipadOS code sample

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#17293] [Q&A] [answered]
Hello
I am looking for an iOS/ ipadOS code sample that would demonstrate how my Avalonia app' could "interop" from C# to platform specific services from iOS, such as StoreKit, or any other "kit" component from Apple.

Any links, pointers ? 

Thank you.

**Comment by thevortexcloud** [comment, marked as answer]
Generally you just use a platform specific TFM, and then you get access to all the official platform specific APIs provided by Microsoft.

https://learn.microsoft.com/en-us/dotnet/standard/frameworks

EG https://learn.microsoft.com/en-us/dotnet/api/storekit?view=xamarin-ios-sdk-12 apparently exists.
